core/tests/test_utils.py

```diff
@@ -1611,6 +1613,22 @@ def save_new_skill_with_defined_schema_versions(
             owner_id, 'New skill created.',
             [{'cmd': skill_domain.CMD_CREATE_NEW}])
 
+    def create_skill_opportunity(
```

Comment:
This actually suggests that we should probably just have a function in opportunity_services whose signature is opportunity_services.create_new_skill_opportunity(skill_id, skill_description) -- I think that that might make more sense, especially if this is only used for skills that don't exist yet. The domain object etc. can be created internally within that function.

It might perhaps also be worth adding a check that such a model does not exist in the datastore yet, and throwing an exception if it does.

Reply:
Done.